puppet/modules/kubernetes/manifests/apiserver.pp

```diff
@@ -171,6 +175,26 @@
     }
   }
 
+  if $audit_enabled {
+
```

Comment:
You need to create the log folder here and make sure it's create before the service is run